### PR TITLE
DO-76: List style fixed for mobile and desktop.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/AppFooter/AppFooter.jsx [hunk 1]
```diff
@@ -18,12 +13,7 @@ import AppFooterMobile from './AppFooterMobile';
 const AppFooter = (props) =>
   (
     <div className={'app-footer'}>
-      <Breakpoint name={'DESKTOP'}>
         <AppFooterDesktop/>
-      </Breakpoint>
-      <Breakpoint name={'BELOW_DESKTOP'}>
-        <AppFooterMobile/>
-      </Breakpoint>
```
Comment (ayushmalik2110) [inline src/components/common/AppFooter/AppFooter.jsx:26]: You are removing the AddFooterMobile then also remove it from import.

Comment (ayushmalik2110) [reply src/components/common/AppFooter/AppFooter.jsx:26]: It looks like you are using the common AppFooter for mobile and desktop. So please remove AppFooterMobile if we are not using it.

File: src/components/common/CardView/CardMoreDetailsArrowIcon/CardMoreDetailsArrowIcon.jsx [hunk 2]
```diff
@@ -15,10 +15,7 @@ const CardMoreDetailsArrowIcon = (props) => {
   const { style: { cardMoreDetailsArrowIconStyle }, onClickMoreDetails } = props;
   return (
     <span onClick={onClickMoreDetails} className={'card-icon'}>
-      <Icon
-        style={cardMoreDetailsArrowIconStyle}
-        icon={faChevronCircleRight}
-      />
+      <svg viewBox="0 0 319.6000061035156 64" aria-labelledby="aqbsi-ionic-ios-more-title" id="si-ionic-ios-more"><title id="aqbsi-ionic-ios-more-title">icon ios-more</title><path d="M159.6 0c-17.7 0-32 14.3-32 32s14.3 32 32 32 32-14.3 32-32-14.3-32-32-32zM32 0C14.3 0 0 14.3 0 32s14.3 32 32 32 32-14.3 32-32S49.6 0 32 0zm255.6 0c-17.7 0-32 14.3-32 32s14.3 32 32 32 32-14.3 32-32-14.3-32-32-32z"></path></svg>
```
Comment (ayushmalik2110) [inline src/components/common/CardView/CardMoreDetailsArrowIcon/CardMoreDetailsArrowIcon.jsx:18]: Create a Svg icon file seperately like IconRight.jsx and use that file here.

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/MemberListDesktop/MemberRow/MemberRow.jsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ const MemberRow = props => {
       }
       <div className={"mbr-list-vw__container__mbr-row__item"} key={'more'} onClick={props.onClickViewDetails}>
         <span>
-         <Icon icon={faChevronCircleRight}/>
+          <svg viewBox="0 0 319.6000061035156 64" aria-labelledby="aqbsi-ionic-ios-more-title" id="si-ionic-ios-more"><title id="aqbsi-ionic-ios-more-title">icon ios-more</title><path d="M159.6 0c-17.7 0-32 14.3-32 32s14.3 32 32 32 32-14.3 32-32-14.3-32-32-32zM32 0C14.3 0 0 14.3 0 32s14.3 32 32 32 32-14.3 32-32S49.6 0 32 0zm255.6 0c-17.7 0-32 14.3-32 32s14.3 32 32 32 32-14.3 32-32-14.3-32-32-32z"></path></svg>
```
Comment (ayushmalik2110) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/MemberListDesktop/MemberRow/MemberRow.jsx:45]: Do not use svg directly create a seperate file for it. Then use that file here.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderDesktopMenu.jsx [hunk 4]
```diff
@@ -12,12 +16,17 @@ import AppHeaderSearchInput from '../../AppHeaderSearchInput/AppHeaderSearchInpu
...
-      <AppHeaderDesktopMenuFilterIcon/>
+
+    <div className={"app-header__menu"}>
       <AppHeaderSearchInput
-        className={"app-header__desktop__menu__item app-header__desktop__menu__search"}
+        className={"app-header__search"}
       />
+      <AppHeaderDesktopMenuFilterIcon/>
+      <AppHeaderDesktopMenuProfileIcon/>
+      <div className={"provider-company-logo"}>
```
Comment (mohitsaladiya) [inline src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderDesktopMenu.jsx:26]: Create a separate component for AppLogo.

File: src/components/routes/Member/AuthenticatedMember/AuthenticatedMember.jsx [hunk 5]
```diff
@@ -17,9 +17,11 @@ class AuthenticatedMember extends React.Component {
     const { config } = this.props;
     if (!isEmpty(config)) {
       return (
-        <div className={"authenticated-mbr-container"}>
+        <div className={'authenticated-mbr__container'}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/AuthenticatedMember.jsx:20]: As per the eslint the correct way to provide className is 
```suggestion
        <div className="authenticated-mbr__container">
```

File: src/components/common/CardView/CardMoreDetailsArrowIcon/CardMoreDetailsArrowIcon.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/CardView/CardMoreDetailsArrowIcon/CardMoreDetailsArrowIcon.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/MemberListDesktop/MemberRow/MemberRow.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/MemberListDesktop/MemberRow/MemberRow.jsx]: done

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderDesktopMenu.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderDesktopMenu.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/AuthenticatedMember.jsx (hunk 5 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/AuthenticatedMember.jsx]: done

**Pull request merged**